js/lesson3/tutorial.md

```diff
@@ -46,7 +50,7 @@ Using jQuery and JavaScript functions, we are going to build a small
 todo list.
 
 Download the files that you will need to work through the example
-[here](download).
+[here](#download-example1). and [example2](#download-example2)
```

Comment:
"_Download the files that you will need to work through the example here. and example2_" doesn't read very well to me.

But I would only download the files for Exercise 1 in the section for exercise 1. The files for exercise 2 should be downloaded from exercise 2.

Also, Exercise 2 still has a link in "_Download the files required to begin working through the example._" pointing to the gist file.